kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/vault/Vault.java

```diff
@@ -88,6 +89,27 @@ public boolean isAvailable() {
         }
     }
 
+    /**
+     * Gets the installed version.
+     *
+     * @return the version
+     */
+    public String getVersionInstalled() {
+        try (var output = new ByteArrayOutputStream();
+                var exec = kubeClient().getClient().pods()
+                        .inNamespace(deploymentNamespace)
+                        .withName(VAULT_POD_NAME)
+                        .writingOutput(output)
+                        .exec("sh", "-c", VAULT_CMD + " version")) {
```

Comment:
There's no need for the shell
```suggestion
                        .exec("VAULT_CMD, "version")) {
```

Reply:
If I remove the shell, the vault command is not recognize and it fails.